packages/ldap-auth-backend/src/provider.ts

```diff
@@ -83,24 +77,15 @@ export class ProviderLdapAuthProvider implements AuthProviderRouteHandlers {
             }
             const { username, password } = req.body;
             const ctx = this.resolverContext;
-            const token = req.cookies?.[this.cookieFieldKey];
+            const token = req.cookies?.[this.cookies.field];
 
             let result: UserIdentityId;
 
             if (username && password) {
                 const { uid } = await this.ldapAuthentication({
-                    ldapOpts: {
-                        url: this.LDAPAuthOpts.url,
-                        tlsOptions: {
-                            rejectUnauthorized:
-                                this.LDAPAuthOpts.rejectUnauthorized,
-                        },
-                    },
-                    userDn: dn`uid=${username},` + this.LDAPAuthOpts.userDn,
-                    userSearchBase: this.LDAPAuthOpts.userSearchBase,
-                    userPassword: password as string,
-                    usernameAttribute: 'uid',
+                    ...this.ldapAuthenticationOptions,
```

Comment:
mmmm, I don't like having all joined, why do you not use a form like `fn(username, password, options)`?

Reply:
I've changed this